test/powershell/Modules/Microsoft.PowerShell.Utility/Test-Json.Tests.ps1

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Test-Json" -Tags "CI" {
+    BeforeAll {
+        $validSchemaJson = @"
```

Comment:
Maybe use just `"` so that the closing `@"` doesn't break the ability to colapse. That or define these string heredocs above the Describe block so they can be unindented.

Reply:
We should place common variables in `BeforeAll`.
It seems more related to the PowerShell VS Code plugin.

Reply:
It's a visual issue too, more than just code collapsing in editors. When scrolling through a file a sudden full-left character causes a short pause in mental tracking. Scrolling this section makes it hard to see where the BeforeAll ends.

I don't think it being in the BeforeAll is that important. If it is before the describe block. so long as the variable name is descriptive, it doesn't change the mental tracking for what the variable represents. The distance between definition and use is already too great and there are quite a few of these. If it was a single one then there may be benefit of definition and use being close. but this many, they already lose meaning beyond variable name when used.

Reply:
@markekraus We use here common patterns which we use in other test files long time.
I agree that some patterns looks bad but I believe they could be discussed in a issue and not block the PR.

Reply:
Normally I would agree, but since this is a new file, it is not beholden to code surrounding it. I would rather see this get merge correct from the beginning rather than have stale issues to resolve it after the fact. as i said, a simple change from `@"` to just `"` will work too if you don't want to move these from the before all block:

```powershell
BeforeAll {
    $validSchemaJson = "
        {
        'description': 'A person',
        'type': 'object',
        'properties': {
            'name': {'type': 'string'},
            'hobbies': {
            'type': 'array',
            'items': {'type': 'string'}
            }
            }
        }
    "
```